modules/MySettings/TabController.php

```diff
@@ -85,13 +85,13 @@ public function get_system_tabs()
                         }
                     }
                     ACLController :: filterModuleList($tabs);
```

Comment:
Not really relevant to your PR, but I noticed this and went 😱 

```suggestion
                    ACLController::filterModuleList($tabs);
```